lib/events/KubeValidate.ts

```diff
@@ -0,0 +1,64 @@
+import {
+    EventFired,
+    EventHandler, Failure,
+    GraphQL,
+    HandleEvent,
+    HandlerContext,
+    HandlerResult, reduceResults, Success,
+    Tags,
+    Value,
+} from "@atomist/automation-client";
+import {fetchCommitForSdmGoal, SdmGoalEvent} from "@atomist/sdm";
+import * as k8 from "kubernetes-client";
+import {getKubeConfig, KubeDeploymentRequest, validateApplication} from "../k8";
+import {SdmGoalSub} from "../typings/types";
+import {CommitForSdmGoal} from "./KubeDeploy";
+
+@EventHandler("get status of k8 cluster", GraphQL.subscription("SdmGoalSub"))
```

Comment:
good job @robertsmargiassi , lets rename and add deployment context...
KubeDeploymentValidate/KubeImageValidate